docs/source/peers/peers.md

```diff
@@ -160,6 +167,22 @@ Later in this section, you'll learn more about the detailed nature of this
 ordering process --- and for a really detailed look at this process see the
 [Transaction Flow](../txflow.html) topic.
 
+### Using Fabric Gateway
+
+Hyperledger Fabric v2.4 introduces the optional Fabric Gateway to simplify client
+application development. Fabric Gateway (the gateway) migrates code for transaction
```

Comment:
I wouldn't use the word `migrates`, that usually implies migrating data. Maybe `delegates` is a better word.
And I wouldn't use the word `code` as that implies the user must provide code. The endorsement management logic is simply shifted from SDK-side to peer-side.

Reply:
ok